### PR TITLE
indices-description

### PR DESCRIPTION
**Pull request by Cephalopterus**
Added an initial draft of the US investing post

### CODE REVIEW

File: us-investing.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# US ETFs and Index Funds
+
+So you want to invest in the US markets then you would notice there are several indices involved in the US markets similar to the Indian counterparts Sensex and Nifty. As with India most of the ETFs and mutual funds you will find in the market will be either tracking these indices (Index funds or ETFs) or be benchmarked to them. I will try to be as objective as possible here though excuse if it feels opinionated in some parts.
```
Comment (crimelabs786) [inline us-investing.md:3]:
```diff
- So you want to invest in the US markets then you would notice there are several indices involved in the US markets similar to the Indian counterparts Sensex and Nifty.
+ Just like Sensex and Nifty in India, US equity markets have plenty of broad-based markets
```

Comment (crimelabs786) [reply us-investing.md:3]: > I will try to be as objective as possible here though excuse if it feels opinionated in some parts. 

Not needed. Remove this line

File: us-investing.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# US ETFs and Index Funds
+
+So you want to invest in the US markets then you would notice there are several indices involved in the US markets similar to the Indian counterparts Sensex and Nifty. As with India most of the ETFs and mutual funds you will find in the market will be either tracking these indices (Index funds or ETFs) or be benchmarked to them. I will try to be as objective as possible here though excuse if it feels opinionated in some parts. 
+
+## Meeting the Indices
+
+As with India there's several indices in the US though we will focus on some of the big ones.
```
Comment (crimelabs786) [inline us-investing.md:7]: Create a bullet-pointed list of 4-5 indices here, and which popular ETFs track them.

Better, make a table (not sure if Discord supports table).

File: us-investing.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+### Dow Jones Industrial Average (DJIA)
+This is the OG index that measures the stock performance of 30 large companies listed on stock exchanges in the United States. The direct comparison to this index is the Indian S&P BSE Sensex 30 index though there are some key differences to the dow and all other indices. This difference is very important and the reason why most people in finance consider this a poor measure of the market. The Dow Jones is not a free float index. What's is a free float methodology? Courtesy: Investopedia, it's defined as
```
Comment (crimelabs786) [inline us-investing.md:10]: Let's avoid language like _OG index_. This FAQ is meant to be consumed by absolute beginners, who might not be familiar with such lingo. Language should be lucid & simple to consume.

We can later create a dank version of these FAQs, that have more Reddit, especially WSB style lingo; where we can add such stuff.

File: us-investing.md (hunk 1 above)
Comment (Cephalopterus) [reply us-investing.md:3]: Got it, added it incase I people are unhappy with comments on the NASDAQ. Many people do not take kindly when told that NASDAQ is not diversified, which objectively, it isn't.

**Review by crimelabs786 — APPROVED** [review]

**Pull request merged**